File: src/config/contact_test.zig
const std = @import("std");
const testing = std.testing;
const contact = @import("contact.zig");

// Pin test — bumping SUPPORT_EMAIL must be a coordinated change across
// every runtime (Zig + website TS + app TS + zombiectl JS + Mintlify
// snippet). The literal IS the contract here; this test fails until
// the matching pin tests in each sibling runtime catch up.
test "SUPPORT_EMAIL pinned to usezombie@agentmail.to" {
    // pin test: literal is the contract
    try testing.expectEqualStrings("usezombie@agentmail.to", contact.SUPPORT_EMAIL);
}

File: ui/packages/website/src/lib/contact.test.ts
import { describe, expect, it } from "vitest";
import { SUPPORT_EMAIL } from "./contact";

// Pin test — bumping SUPPORT_EMAIL must be a coordinated change across
// every runtime (Zig + website TS + app TS + agentsfleet JS + Mintlify
// snippet). The literal IS the contract here.
describe("SUPPORT_EMAIL pinned (regression — mirror src/config/contact_test.zig)", () => {
  it("resolves to usezombie@agentmail.to", () => {
    // pin test: literal is the contract
    expect(SUPPORT_EMAIL).toBe("usezombie@agentmail.to");
  });
});
